gravitee-am-reporter/gravitee-am-reporter-mongodb/src/main/java/io/gravitee/am/reporter/mongodb/audit/MongoAuditReporter.java

```diff
@@ -240,6 +242,7 @@ private void initIndexes() {
             Map<Document, IndexOptions> indexes = new HashMap<>();
             indexes.put(new Document(FIELD_REFERENCE_TYPE, 1).append(FIELD_REFERENCE_ID, 1).append(FIELD_TIMESTAMP, -1), new IndexOptions().name(INDEX_REFERENCE_TIMESTAMP_NAME));
             indexes.put(new Document(FIELD_REFERENCE_TYPE, 1).append(FIELD_REFERENCE_ID, 1).append(FIELD_TYPE, 1).append(FIELD_TIMESTAMP, -1), new IndexOptions().name(INDEX_REFERENCE_TYPE_TIMESTAMP_NAME));
+            indexes.put(new Document(FIELD_REFERENCE_TYPE, 1).append(FIELD_REFERENCE_ID, 1).append(FIELD_TYPE, 1).append(FIELD_STATUS, 1).append(FIELD_TIMESTAMP, -1), new IndexOptions().name(INDEX_REFERENCE_TYPE_STATUS_SUCCESS_TIMESTAMP_NAME).partialFilterExpression(new Document(FIELD_STATUS, new Document("$eq", "SUCCESS"))));
```

Comment:
Do we have some indexes to set for the JDBC (SQL) implementation ?

Reply:
Dont think we can provide such. The query for JDBC requires more in depth analysis as it has a few JOINs. 
Even though I added index, it didn't make a query faster nor didn't remove SeqScans.

![image](https://github.com/gravitee-io/gravitee-access-management/assets/158267130/7929477f-0caf-41b2-93ef-093cfb1bdc59)
 
I would leave JDBCs indexes intact